airflow/include/dagintegritytest.go

```diff
@@ -22,55 +23,70 @@ def suppress_logging(namespace):
     finally:
         logger.disabled = old_value
 
+
```

Comment:
instead of hardcoding it inside go file, let's move it to python file, so we can run `black` and read this file from go source code into variable.

@neel-astro thoughts?

Reply:
Agree with you, I will create a separate issue for that, and work on it outside of this PR if that is fine